### DISCUSSION TITLE
Version conflict for eslint-plugin-unicorn

### DISCUSSION THREAD

**Discussion by fhg-isi** [renovatebot/renovate#25517] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I use renovate with following GitHub workflow action

https://github.com/fraunhofer-isi/micat/blob/main/.github/workflows/renovate.yml

renovate updated the dependencies and now I have a version conflict.

 eslint-recommended/eslint-config@16.0.3 requires eslint-plugin-unicorn@48.0.0

but renovate chose 49.0.0

### Have you created a minimal reproduction repository?

No. But issue can be seen here:
https://github.com/fraunhofer-isi/micat/actions/runs/6706850723/job/18224128304

**Comment by rarkins** [comment]
Define in technical terms what you mean by "npm works correctly". What command/code could Renovate run to determine this?

**Reply by fhg-isi** [reply]
Ah. Sorry. I have been confused by the warning "package was not found". The actual issue that caused the failing pipeline seems to be a different one. 
For the warning I created some SO question: https://stackoverflow.com/questions/77396525/npm-shows-warning-the-following-package-was-not-found-and-will-be-installed

**Comment by rarkins** [comment, marked as answer]
> Now another workflow action fails with following issue:
> 
> `npm WARN exec The following package was not found and will be installed: next@14.0.1`

I don't think that's what causes the failure. It looks to be a peer dependencies problem, and no amount of waiting is likely to fix that.

**Comment by fhg-isi** [comment]
I specified an explicit version for eslint-plugin-unicorn

{
      // eslint-recommended/eslint-config@16.0.3 requires eslint-plugin-unicorn@"48.0.0"
      "matchPackageNames": ["eslint-plugin-unicorn"],
      "allowedVersions": "48.0.0"
    }
